googleapis/google-cloud-node
# migrate code from googleapis/nodejs-private-catalog

- feat: add initial lib stub
- chore: update package name (#6)
- chore(deps): update dependency sinon to v11 (#4)
- docs: add initial samples (#2)
- chore: release 1.0.0 (#1)
- build(nodejs): name should be @google-cloud/private-cataglog (#20)
- fix(deps): require google-gax v2.17.0 (#17)
- chore: release 1.0.1 (#21)
- fix(deps): google-gax v2.17.1 (#22)
- chore: release 1.0.2 (#23)
- fix: Updating WORKSPACE files to use the newest version of the Typescript generator. (#24)
- chore: release 1.0.3 (#25)
- chore(nodejs): update client ref docs link in metadata (#30)
- fix(deps): google-gax v2.24.1 (#32)
- chore: release 1.0.4 (#33)
- feat: turns on self-signed JWT feature flag (#34)
- chore: release 1.1.0 (#35)
- chore: disable renovate dependency dashboard (#1194) (#36)
- fix(build): switch primary branch to main (#37)
- chore: release 1.1.1 (#38)
- docs(samples): add auto-generated samples for Node with api short name in region tag (#43)
- build(node): update deps used during postprocessing (#1243) (#46)
- chore: fix the wrong post processor image (#49)
- chore(deps): update dependency @types/node to v16 (#48)
- chore(cloud-rad): delete api-extractor config (#51)
- docs(samples): add example tags to generated samples (#52)
- chore(deps): update dependency sinon to v12 (#50)
- build: add generated samples to .eslintignore (#53)
- docs(node): support "stable"/"preview" release level (#1312) (#56)
- chore: add api_shortname and library_type to repo metadata (#55)
- docs(badges): tweak badge to use new preview/stable language (#1314) (#58)
- test(nodejs): remove 15 add 16 (#1322) (#60)
- build: update copyright year to 2022 (#62)
- chore: update v2.12.0 gapic-generator-typescript (#64)
- chore(deps): update dependency sinon to v13 (#66)
- docs(nodejs): version support policy edits (#1346) (#68)
- docs(samples): include metadata file, add exclusions for samples to handwritten libraries (#69)
- build(node): update client library version in samples metadata (#1356) (#79)
- chore(deps): update dependency @types/mocha to v9 (#80)
- chore(deps): update dependency sinon to v14 (#83)
- build!: update library to use Node 12 (#85)
- chore(main): release 2.0.0 (#86)
- chore(deps): update dependency jsdoc-region-tag to v2 (#89)
- chore(deps): update dependency jsdoc-fresh to v2 (#88)
- fix(docs): describe fallback rest option (#90)
- chore(main): release 2.0.1 (#91)
- chore(deps): update dependency linkinator to v4 (#95)
- chore: remove unused proto imports (#96)
- fix: better support for fallback mode (#97)
- fix: change import long to require (#98)
- chore(main): release 2.0.2 (#99)
- fix: do not import the whole google-gax from proto JS (#1553) (#101)
- fix: allow passing gax instance to client constructor (#102)
- chore(main): release 2.0.3 (#103)
- fix: preserve default values in x-goog-request-params header (#104)
- test: use fully qualified request type name in tests (#107)
- fix(deps): use google-gax v3.5.2 (#111)
- chore(deps): update dependency jsdoc to v4 (#113)
- fix: regenerated protos JS and TS definitions (#114)
- chore(deps): update dependency @types/node to v18 (#109)
- chore(main): release 2.0.4 (#105)
- build: add release-please config, fix owlbot-config

Thank you for opening a Pull Request! Before submitting your PR, there are a few things you can do to make sure it goes smoothly:

Fixes #<issue_number_goes_here> 🦕
